src/transform/ir_proto_to_datalog/utils/access_path_test.cc

```diff
@@ -0,0 +1,159 @@
+#include "access_path.h"
+
+#include "src/common/testing/gtest.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/str_split.h"
+
+#include <string>
+
+namespace raksha::transform::utils {
+
+// Access paths are easier to read and write in their string form, so this is
+// very useful for creating readable test inputs.
+static absl::StatusOr<utils::AccessPath> make_access_path_from_string(
```

Comment:
Would it be worth adding this to core `AccessPath` data structure?

Reply:
Perhaps. It does seem like it could be useful there, but I hadn't run into a use for it yet, so I was hesitant to add something to the implementation to serve the tests. Happy to move it if you think it would be better there though.

Reply:
With the implementation of FieldSelector, it's clear that the logic here is a bit provincial to that particular selector. I'm going to hold off on moving this to the implementation until it's a bit more general.